db/migrate/20151230082305_add_container_event_ems_ref.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerEventEmsRef < ActiveRecord::Migration
+  def change
+    add_column :event_streams, :ems_ref, :string
```

Comment:
@dkorn if `emd_ref` was needed I expect it to be in the table already. If not then I don't think there's anything special in containers (vs VMs) to require it.

Are you sure there's nothing similar already?